tests/test_bot.py

```diff
@@ -395,23 +395,42 @@ def test_send_message(self):
                                         short=True)])])
         self.bot.send_message(message)
 
-        attachments = """[{"fallback": "fallback", "color": "#36a64f",
- "author_name": "John Doe", "author_link": "https://example.com/author",
- "author_icon": "https://example.com/author.jpg", "title": "Title",
- "title_link": "https://example.com", "text":
- "Go to Example Website", "image_url": "https://example.com/image.jpg",
- "thumb_url": "https://example.com/thumb.jpg", "fields": [{"title":
- "Established", "value": "2008", "short": false},
- {"title": "Users", "value": "27 Million",
- "short": true}], "footer": "Footer text", "footer_icon":
- "https://example.com/footer.jpg"}]""".replace('\n', "")
-
-        self.bot.slack_client.api_call.assert_called_with('chat.postMessage',
-                                                          channel='channel_id',
-                                                          as_user=True,
-                                                          attachments=(
-                                                           attachments),
-                                                          text=None)
+        attachments = """
+        [
+            {
+                "fallback":"fallback",
+                "color":"#36a64f",
+                "author_name":"John Doe",
+                "author_link":"https://example.com/author",
+                "author_icon":"https://example.com/author.jpg",
+                "title":"Title",
+                "title_link":"https://example.com",
+                "text":"Go to Example Website",
+                "image_url":"https://example.com/image.jpg",
+                "thumb_url":"https://example.com/thumb.jpg",
+                "fields":[
+                    {
+                        "title":"Established",
+                        "value":"2008",
+                        "short":false
+                    },
+                    {
+                        "title":"Users",
+                        "value":"27 Million",
+                        "short":true
+                    }
+                ],
+                "footer":"Footer text",
+                "footer_icon":"https://example.com/footer.jpg"
+            }
+        ]
+"""
+        self.bot.slack_client.api_call.assert_called_with(
+                'chat.postMessage',
+                channel='channel_id',
+                as_user=True,
+                attachments=json.dumps(json.loads(attachments)),
```

Comment:
This might appear odd, but `json.loads()` outputs to JSON which uses double quotes. The test is expecting single quotes to be used, despite the input actually using standard JSON. `json.dumps()` outputs using single quotes.